node/core/pvf/src/prepare/worker.rs

```diff
@@ -270,7 +363,9 @@ pub fn worker_entrypoint(socket_path: &str) {
 					);
 					async_std::fs::write(&dest, &compiled_artifact).await?;
 
-					Ok(())
+					// TODO: We are now sending the CPU time back, changing the expected interface. Do
+					// we need to account for this breaking change in any way?
```

Comment:
The data that the host expects from the child has changed. I think it's fine but wanted to double-check that this wouldn't cause breakage.